extensions/Worker.Extensions.Tables/src/Config/TablesBindingOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+using Azure.Data.Tables;
+using Azure.Storage.Blobs;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Tables.Config
+{
+    internal class TablesBindingOptions
+    {
+        public string? ConnectionString { get; set; }
+
+        public Uri? ServiceUri { get; set; }
+
+        public TokenCredential? Credential { get; set; }
+
+        public TableClientOptions? TableClientOptions { get; set; }
+
+        internal TableServiceClient CreateClient()
```

Comment:
Let's implement a client cache similar to what we have in CosmosDBBindingOptions.cs

Reply:
done!